test/functional/apps/discover/_data_grid_context.ts

```diff
@@ -78,5 +86,26 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       }
       expect(disabledFilterCounter).to.be(TEST_FILTER_COLUMN_NAMES.length);
     });
+
+    // Skipped because there is a bug in the data grid row expansion
+    it.skip('navigates to context view from embeddable', async () => {
```

Comment:
👍

Reply:
should now be safe to unskip, since it was resolved in #92999